### ISSUE TITLE
Running cargo test in windows

### ISSUE THREAD

**Issue by fabricereix** [Orange-OpenSource/hurl#128]
Do use /tmp in unit tests
```
---- runner::body::tests::test_body_file stdout ----
thread 'runner::body::tests::test_body_file' panicked at 'couldn't create /tmp/data.bin: Os { code: 3, kind: NotFound, message: "Le chemin d’accès spécifié est introuvable." }', packages\hurl\src\runner\body.rs:98:29
```

**Comment by fabricereix** [comment]
only the doc test in hurl_file.rs does not pass

**Comment by fabricereix** [comment]
fix in #130
